File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/androidx/ArchitectureActivity.java
package com.jdroid.android.sample.androidx;

import com.jdroid.android.activity.FragmentContainerActivity;
import com.jdroid.android.sample.database.room.RoomFragment;

import androidx.fragment.app.Fragment;

public class ArchitectureActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return ArchitectureFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/androidx/ArchitectureActivity.kt
package com.jdroid.android.sample.androidx

import androidx.fragment.app.Fragment
import com.jdroid.android.activity.FragmentContainerActivity

class ArchitectureActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return ArchitectureFragment::class.java
    }
}
